lua/utils/init.lua

```diff
@@ -146,6 +144,18 @@ function utils.join_paths(...)
   return result
 end
 
+function utils.write_file(path, txt, flag)
```

Comment:
is this used ?

Reply:
Not in this iteration, but I used it to write all the files in lsp/providers.